Share/Share/ShareViewModel.ts

```diff
@@ -224,38 +224,7 @@ class ShareViewModel extends Accessor {
       y,
       spatialReference
     });
-    // Use pointToConvert to project point. Once projected, pass point to generate the share URL parameters
-    const point = await this._processPoint(centerPoint);
-    return this._generateShareUrlParams(point);
-  }
-
-  private async _processPoint(point: Point): Promise<__esri.Point> {
-    const { isWGS84, isWebMercator } = point.spatialReference;
-    // If spatial reference is WGS84 or Web Mercator, use longitude/latitude values to generate the share URL parameters
-    if (isWGS84 || isWebMercator) {
-      return point;
-    }
-    // Check if client side projection is not supported
-    if (!projection.isSupported()) {
```

Comment:
I think the only thing that should be removed is projection.isSupported. I think the rest of the logic needs to be there right?